packages/reactstrap-validation-date/src/AvDateRange.js

```diff
@@ -348,13 +401,84 @@ export default class AvDateRange extends Component {
       this.props.end.name
     ).getViewValue();
 
-    if (start && !end) {
+    const hasStart = start && start !== '';
+    const hasEnd = end && end !== '';
+
+    if (hasStart && !hasEnd) {
       return 'Both start and end date are required.';
     }
 
     return true;
   };
 
+  renderDateRanges = () => {
+    const { ranges: propsRanges } = this.props;
+    const { startValue, endValue, format } = this.state;
+
+    console.log('propsRanges', propsRanges);
```

Comment:
let's remove

Reply:
Hmmm, wondering why the linter didn't blow up on this.

https://github.com/Availity/availity-react/pull/246/commits/d86a3fd4598313595f0d82fb7048b57375962d26